challenge-030/creewick/ch-1.pl

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/perl
```

Comment:
`#!/usr/bin/env perl` is better. On some systems, interpreter may be situated in another place. And perlbrew won't work with such shebang